src/client/components/results/Movie.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class Movie extends React.Component {
```

Comment:
@vitkuz this component must be stateless functional one. Again, this error must be caught by linter.

Reply:
done